models/utils.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  // https://github.com/Vincit/objection.js/issues/192
+  findOrCreate: async (Model, data) => {
+    const fetched = await Model.query().where(data)
```

Comment:
@duranmla How is this supposed to work? If we are persisting data the same way we are querying it, how it finds the record? Given that we only need to find by `externalId`.

Instead of doing this I thnk it's better to have the method inside the User model. Implement it in a case by case basis and if it's needed make the abstraction